### PR TITLE
Update midfeedback form

### LINKED ISSUE #2153: Update Mid Feedback Form
Update MidFeedbackForm.js to use Formik, Yup, and Bootstrap as documented in issue #2136. These changes have been made to HowDidItStartForm.js, you can them here - #2138.  

Form components were created as part of PR #2138. It might be a good idea to replace all of the Chakra-UI layout components here.

### PR DESCRIPTION
**Pull request by DianeLiu2019**
Fixes #2153(issue)

# Description
update midFeedbackForm.js by using formik, bootstrap
> Please include a summary of the change.

# Any new packages installed?

> Give details

# Required followup work

> Is there anything related to this that still needs to be done (ex: documentation changes).

# Checklist:

- [ ] I have updated the azurescript.sh with any **new environment variables** and added them to the appsettings
- [ ] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [ ] I have added and needed tests for my changes (in particular for new screens)
- [ ] I have added a comment to any confusing code
- [ ] I have added documentation to any modified front-end code. (Or added missing documentation)

### CODE REVIEW

**Comment by ying-chen-cds** [comment]
The submit button for mid feedback looks totally different than the original one. Is this what we want?
**Original:**
original

**After refactoring:**
new

File: f2/src/forms/MidFeedbackForm.js [hunk 1]
```diff
@@ -111,58 +126,89 @@ export const MidFeedbackForm = (props) => {
...
-                    onSubmit={handleSubmit}
-                    shouldWrapChildren
-                    spacing={6}
-                  >
-                    {showWarning ? (
-                      <Alert status="error">
-                        <Trans id="finalFeedback.warning" />
-                      </Alert>
-                    ) : null}
+              >
```
Comment (ying-chen-cds) [inline f2/src/forms/MidFeedbackForm.js:129]: The validation message should not just be removed.

**Comment by ying-chen-cds** [comment]
temp
"adaptive technology" should move up.

**Comment by DianeLiu2019** [comment]
Good catch. I still working on it. It is not ready for review yet.

File: f2/src/theme/style.css [hunk 2]
```diff
@@ -35,17 +35,20 @@ body {
   margin-bottom: 0.25rem;
   line-height: 1;
   margin-left: 0px;
+  max-width: 600px;
 }
 
 .form-question {
   padding-bottom: 1rem;
+  display: grid;
```
Comment (justinr86) [inline f2/src/theme/style.css:43]: There is no need for display grid, that is what Bootstrap is for.

File: f2/src/theme/style.css [hunk 3]
```diff
@@ -35,17 +35,20 @@ body {
...
 }
 
 .form-helper-text {
   margin-top: 0.25rem;
   line-height: 1.25;
   font-size: 1rem;
   margin-left: 0px;
+  max-width: 600px;
```
Comment (justinr86) [inline f2/src/theme/style.css:51]: Why add a max width?

File: f2/src/theme/style.css [hunk 4]
```diff
@@ -292,6 +295,8 @@ body {
 .custom-control-label {
   padding-left: 1.5rem;
   font-size: 1.25rem;
+  line-height: 1.5rem;
```
Comment (justinr86) [inline f2/src/theme/style.css:298]: This makes the text non-responsive. On smaller screens the problem is worse, I think a smaller line-height is a good start but the padding-top needs to be removed in favor of another rule.

File: f2/src/forms/MidFeedbackForm.js [hunk 5]
```diff
@@ -1,60 +1,61 @@
 /** @jsx jsx */
 import React, { useState } from 'react'
+import PropTypes from 'prop-types'
 import { jsx } from '@emotion/core'
 import { H1, H2 } from '../components/header'
```
Comment (justinr86) [inline f2/src/forms/MidFeedbackForm.js:5]: Should use the <P> component from f2\src\components\formik\paragraph\index.js. You may have to style a new component for H1 and H2, let me know if you need help.

File: f2/src/forms/MidFeedbackForm.js [hunk 6]
```diff
@@ -1,60 +1,61 @@
...
 import { useLocation } from 'react-router-dom'
-import { Stack, Box } from '@chakra-ui/core'
-import { containsData } from '../utils/containsData'
-import { Button } from '../components/button'
-import { TextArea } from '../components/text-area'
+import { Trans } from '@lingui/macro'
+import { Form, Container, Row } from 'react-bootstrap'
+import { Formik, FieldArray, Field } from 'formik'
+import { CheckBox } from '../components/formik/checkbox'
 import { InfoCard } from '../components/container'
```
Comment (justinr86) [inline f2/src/forms/MidFeedbackForm.js:15]: Should use an Alert from f2\src\components\formik\alert\index.js. You may need to style a new component for this, let me know if you need help.

File: f2/src/components/formik/button/index.js [hunk 7]
```diff
@@ -90,6 +94,14 @@ export const UploadButton = (props) => {
   )
 }
 
+export const FeedbackButton = (props) => {
+  return (
+    <Button type="submit" className="button feedback-button">
```
Comment (justinr86) [inline f2/src/components/formik/button/index.js:99]: The class name **feedback-button** is never defined. The result is a default Bootstrap button.

Comment (ying-chen-cds) [reply f2/src/components/formik/button/index.js:99]: I defined it in refactoring final feedback form, this code is probably from there.

**Pull request merged**